modules/imap/output_modules.php

```diff
@@ -171,6 +171,11 @@ protected function output() {
                 $this->html_safe($this->get('msg_list_path')),
                 $this->html_safe($this->get('msg_text_uid'))
             );
+            /* $reply_args_forward_attachment = sprintf('&amp;list_path=%s&amp;uid=%d',
```

Comment:
Please delete this comment if not needed

Reply:
I modified this function and made fixes